### PR TITLE
Create Card component and Update UI

### PR DESCRIPTION
**Pull request by shoryakaushik**
1. Updated Card UI
2. Made a separate Card Component to be used anywhere within the app.

### CODE REVIEW

**Comment by suubh** [comment]
Nice Work .
Please apply some of these required  changes :
- Change the background color of cards to white.
- Keep the button style as it was earlier.(border-radius:10px and background-color:blue)

File: src/components/card/styles.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.post{
+    width: 300px;
+    height: 300px;
+    background-color: #46da95;
```
Comment (suubh) [inline src/components/card/styles.css]: background-color: #FFFFFF;

File: src/components/card/styles.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    border-radius: 50%;
+    width: 50%;
+    height: 50%;
+    padding-top: 10px;
+  }
+  
+  button {
+    transition: transform 0.33s !important;
+    border-color: #ea4d67 ;
+    background-color: #ea4d67;
```
Comment (suubh) [inline src/components/card/styles.css]: background-color: #61dafb;

File: src/components/card/styles.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    border-radius: 2px;
```
Comment (suubh) [inline src/components/card/styles.css]: border-radius: 10px;

**Review by suubh — CHANGES_REQUESTED** [review]

File: src/components/card/styles.css (hunk 1 above)
Comment (shoryakaushik) [reply src/components/card/styles.css]: Fixed

File: src/components/card/styles.css (hunk 2 above)
Comment (shoryakaushik) [reply src/components/card/styles.css]: Fixed

File: src/components/card/styles.css (hunk 3 above)
Comment (shoryakaushik) [reply src/components/card/styles.css]: Fixed

**Comment by shoryakaushik** [comment]
@suubh Changes are done, you can review again.

**Pull request merged**